src/components/TextArea/TextArea.driver.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './TextArea.st.css';
+import {
+  TEXT_AREA_SUCCESS_ICON,
+  TEXT_AREA_DATA_HOOK,
+  TEXT_AREA_ERROR_ICON,
+  TEXT_AREA_ERROR_DESCRIPTION,
+} from './dataHooks';
+import { TextAreaTheme } from './TextArea';
+
+export interface TextAreaDriver extends BaseUniDriver {
+  value(): Promise<string>;
+  error(): Promise<boolean>;
+  success(): Promise<boolean>;
+  isMobile(): Promise<boolean>;
+  disabled(): Promise<boolean>;
+  placeholder(): Promise<string>;
+  theme(): Promise<TextAreaTheme>;
+  typeText(text: string): Promise<any>;
+}
+
+function textArea(base: UniDriver) {
+  return base.$(`[data-hook="${TEXT_AREA_DATA_HOOK}"]`);
+}
+
+function successIcon(base: UniDriver) {
+  return base.$(`[data-hook="${TEXT_AREA_SUCCESS_ICON}"]`);
+}
+
+function errorIcon(base: UniDriver) {
+  return base.$(`[data-hook="${TEXT_AREA_ERROR_ICON}"]`);
+}
+
+function errorText(base: UniDriver) {
+  return base.$(`[data-hook="${TEXT_AREA_ERROR_DESCRIPTION}"]`);
+}
+
+export const textAreaDriverFactory = (base: UniDriver): TextAreaDriver => {
+  const stylableUtil = new StylableUnidriverUtil(style);
```

Comment:
Great usage of data hooks!
I would also like if you could use `data-*` attributes instead of using the `StyleableUnidriverUtil`.
Because using `StyleableUnidriverUtil` requires us to import the `*.st.css` file into the driver, and this can cause problems in the tests.
So for instance, instead of using: ```stylableUtil.hasStyleState(base, 'disabled')```, we can do:
```
base.$(`[${TEXT_AREA_DATA_DISABLED}="true"]`)
```

Reply:
I know it came as part of the generator, it's something that we need to fix